pkg/liqo-controller-manager/networking/ip-controller/ip_controller.go

```diff
@@ -114,7 +115,7 @@ func (r *IPReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Re
 		}
 
 		// Remove finalizer from the IP resource if present.
-		if controllerutil.ContainsFinalizer(&ip, ipamIPFinalizer) {
+		if controllerutil.ContainsFinalizer(&ip, ipamIPFinalizer) && !utils.IsPreinstalledResource(&ip) {
```

Comment:
```suggestion
		if controllerutil.ContainsFinalizer(&ip, ipamIPFinalizer) {
```

The preinstalled resources should not have the finalizer, but I would not check this if, for some strange reason,n they have